### PR TITLE
Faster e2e implementation

### PR DESCRIPTION
**Pull request by vkuznecovas**
New:
 mage teste2ebasic  43.23s user 18.09s system 21% cpu **4:45.21** total

Old:
mage teste2ebasic  29.06s user 12.90s system 8% cpu **8:42.82** total

As a bonus, the mac now stays cool to the touch when running e2e. The savings will be smaller on linux, but I expect them to still be noticeable.

### CODE REVIEW

File: e2e/runner.go [hunk 1]
```diff
@@ -69,13 +69,13 @@ func (r *Runner) Test(providerHost, consumerHost string) error {
...
 	err := r.compose("run", "go-runner",
-		"go", "test", "-v", "./e2e/...", "-args",
-		"--provider.tequilapi-host", providerHost,
-		"--provider.tequilapi-port=4050",
-		"--consumer.tequilapi-host", consumerHost,
-		"--consumer.tequilapi-port=4050",
-		"--geth.url=ws://ganache:8545",
-		"--consumer.services", r.services,
+		"/usr/bin/test", "-test.v",
+		"-provider.tequilapi-host", providerHost,
```
Comment (zolia) [inline e2e/runner.go:73]: should be no difference single vs double dashes. Postfix notation usually prefers double dashes.

File: e2e/runner.go [hunk 2]
```diff
@@ -133,10 +135,15 @@ func (r *Runner) Init() error {
 		return errors.Wrap(err, "starting mysterium-api failed!")
 	}
 
+	log.Info().Msg("Force rebuilding go runner")
+	if err := r.compose("build", "go-runner"); err != nil {
+		return errors.Wrap(err, "building go-runner!")
```
Comment (zolia) [inline e2e/runner.go:140]: we use %w wrapping now. And should sound as an error.

File: e2e/gorunner/Dockerfile.precompiled [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.6
```
Comment (soffokl) [inline e2e/gorunner/Dockerfile.precompiled:1]:
```suggestion
FROM alpine:3.11
```

File: mobile/mysterium/openvpn_connection_setup.go [hunk 4]
```diff
@@ -27,11 +27,11 @@ import (
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
 
-	"github.com/mysteriumnetwork/go-openvpn/openvpn3"
 	"github.com/mysteriumnetwork/node/core/connection"
 	"github.com/mysteriumnetwork/node/core/ip"
 	"github.com/mysteriumnetwork/node/core/port"
 	"github.com/mysteriumnetwork/node/identity"
+	openvpn3 "github.com/mysteriumnetwork/node/mobile/mysterium/openvpn3"
```
Comment (soffokl) [inline mobile/mysterium/openvpn_connection_setup.go:34]: https://github.com/mysteriumnetwork/node/pull/2084

File: bin/docker/alpine-prebuilt/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.6
```
[the remark this answers is not in the document]
Comment (anjmao) [reply bin/docker/alpine-prebuilt/Dockerfile:1]: If we change version here when release alpine image should use the same version too.

File: mobile/mysterium/openvpn_connection_setup.go (hunk 4 above)
Comment (anjmao) [reply mobile/mysterium/openvpn_connection_setup.go:34]: You can revert it now, and I will check why it's not reverting back.

Comment (vkuznecovas) [reply mobile/mysterium/openvpn_connection_setup.go:34]: my ide seems to be flipping them, i'll try and force a proper one

File: bin/docker/alpine-prebuilt/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.11
```
Comment (anjmao) [inline bin/docker/alpine-prebuilt/Dockerfile:1]: Update bin/docker/alpine version too.

**Pull request merged**